examples/ex1.cpp

```diff
@@ -232,15 +232,11 @@ int main(int argc, char *argv[])
       int  visport   = 19916;
       socketstream sol_sock(vishost, visport);
       sol_sock.precision(8);
-      sol_sock << "solution\n" << *mesh << x << flush;
+      sol_sock << "solution\n" << mesh << x << flush;
    }
 
    // 15. Free the used memory.
-   delete a;
-   delete b;
-   delete fespace;
-   if (order > 0) { delete fec; }
-   delete mesh;
+   delete fec;
```

Comment:
The condition for deleting `fec` should not be changed.

Reply:
The previous condition was ignoring the `new` line 136 though.